backend/danswer/background/file_deletion.py

```diff
@@ -0,0 +1,6 @@
+from danswer.background.utils import interval_run_job
+from danswer.connectors.file.utils import clean_temp_files
+
+
+if __name__ == "__main__":
```

Comment:
Can we add a comment on the logic here, like how the logic prevents deleting files that haven't yet been indexed etc

Reply:
or alternatively just call it clean_processed_files or something

Reply:
Sure, added a more descriptive name